### PR TITLE
Closes #142 Add support for UniversalLinks, closes #143 Support POP and PDP web URLs

### LINKED ISSUE #142: Add support for UniversalLinks
In order to handle URLs from the Sunrise web, UniversalLinks support needs to be added to the project, accompanied by adding the _apple-app-site-association_ file to the web.

### LINKED ISSUE #143: Support POP and PDP web URLs
- If the user taps on a POP link from their messaging app / email client, the Sunrise iOS app should present the POP with the search text pre-populated.
At this point, the app should only take the value from the `q` parameter (e.g https://demo.commercetools.com/en/search?q=jeans). Once we add facets / filters, the URL parsing logic should be updated to support them as well.

- If the user taps on a PDP link from their messaging app / email client, the Sunrise iOS app should present the PDP for the product SKU contained in the link (e.g https://demo.commercetools.com/en/peuterey-downjacket-anita-B351-olive-M0E20000000E4RB.html)

### PR DESCRIPTION
**Pull request by nikola-mladenovic**
TestFlight build will be available right after we add _apple-app-site-association_ file to the web.

### CODE REVIEW

File: Sunrise/AppDelegate.swift
Comment (cneijenhuis) [inline Sunrise/AppDelegate.swift]: Can we add an example URL here and below?

**Review by cneijenhuis — COMMENTED** [review]
Waiting for TestFlight for final approval 😉

**Comment by nikola-mladenovic** [comment]
We have the _apple-app-site-association_ live, and after doing some edge-case clean-up, TestFlight build # 78 is being processed, and should be available for testing shortly.

File: Sunrise/AppDelegate.swift
Comment (nikola-mladenovic) [reply Sunrise/AppDelegate.swift]: Added examples for both 👍

**Review by cneijenhuis — APPROVED** [review]
We could also give some feedback here if the SKU can not be found, but probably it's less critical/likely.

**Comment by nikola-mladenovic** [comment]
Good point, will add an error message in case the product was not found (as a part of the PR #145 )

**Pull request merged**
